tests/utils/itutils.go

```diff
@@ -20,7 +20,7 @@ import (
 )
 
 // Deis points to the CLI used to run tests.
-var Deis = "deis "
+var Deis = os.Getenv("DEIS_BINARY")
```

Comment:
this will break local environments that do not have $DEIS_BINARY set (I don't run test-integration.sh locally, just boot up my own cluster then run `make -C tests test-full`). Would you mind changing this to:

```
var Deis = os.Getenv("DEIS_BINARY")
if Deis == "" {
    Deis = "deis "
}
```

While we're at it, we should remove the space requirement in $DEIS_BINARY and just update the one reference to this variable :)